src/Preprocessing.py

```
@@ -0,0 +1,67 @@
+import os
+import cv2
+from PIL import Image, ExifTags
+import numpy as np
+
+def auto_rotate_if_landscape(img):
```

Flake8 (7.2.0)

[error] expected 2 blank lines, found 1

(E302)